### ISSUE TITLE
Use a response file in WinMDExp task

### ISSUE THREAD

**Issue by rainersigwald** [Microsoft/msbuild#2140]
An internal team ran into a problem that caused `WinMDExp` to give the error

```
1>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(2352,7): warning MSB6002: The command-line for the "WinMDExp" task is too long. Command-lines longer than 32000 characters are likely to fail. Try reducing the length of the command-line by breaking down the call to "WinMDExp" into multiple calls with fewer parameters per call.
1>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(2352,7): error MSB6003: The specified task executable "winmdexp.exe" could not be run. The filename or extension is too long
1>Done executing task "WinMDExp" -- FAILED.
```

`WinMDExp` is a `ToolTaskExtension`, so the error is happening when it calls `CreateProcess` on `winmdexp.exe` with a very long command line.

Fortunately, `winmdexp.exe` can take a response file (according to its help), so this can be avoided entirely by calling `ToolTaskExtension.AddResponseFileCommands()` instead of `AddCommandLineCommands()`.

**Comment by rainersigwald** [comment]
Reactivating: `AddResponseFileCommands` emits a response file with a single line, which is unacceptable to `winmdexp.exe`.

**Comment by rainersigwald** [comment]
Closed with #2216.
